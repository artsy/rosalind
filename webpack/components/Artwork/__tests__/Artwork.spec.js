import React from 'react'
import renderer from 'react-test-renderer'
import Artwork from 'components/Artwork'

it('renders correctly', () => {
  const artworkFixture = {
    title: 'title',
    images: [
      { image_urls: { normalized: 'soup-can.jpg' } }
    ],
    artist: { name: 'andy warhol' }
  }
  const tree = renderer.create(<Artwork {...artworkFixture} />).toJSON()
  expect(tree).toMatchSnapshot()
})
