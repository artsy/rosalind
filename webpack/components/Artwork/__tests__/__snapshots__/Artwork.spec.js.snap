exports[`test renders correctly 1`] = `
<div
  className="Artwork">
  <h1>
    title
  </h1>
  <h2>
    andy warhol
  </h2>
  <img
    src="soup-can.jpg" />
</div>
`;
